hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/tools/offlineImageViewer/TestOfflineImageViewer.java

```diff
@@ -838,7 +838,7 @@ public void testReverseXmlWithoutSnapshotDiffSection() throws Throwable {
       writer.println("<?xml version=\"1.0\"?>");
       writer.println("<fsimage>");
       writer.println("<version>");
-      writer.println("<layoutVersion>-67</layoutVersion>");
+      writer.println("<layoutVersion>-65</layoutVersion>");
```

Comment:
The change itself is good. Would you create a separate issue and separate PR for this fix?

Reply:
Fixed by https://issues.apache.org/jira/browse/HDFS-16546. Thanks